opennms-doc/releasenotes/src/asciidoc/releasenotes/whatsnew.adoc

```diff
@@ -68,6 +68,7 @@ A number of provisioning, polling, and web UI changes have been made to accomoda
 * Flow DNS enrichment has been enhanced to improve performance.
 * Aggregate flow queries now support resolving hostnames if available.
 * Sink RPC APIs have been enhanced to expose metrics to Karaf for tracing and performance tuning.
+* The OpenConfig integration now supports gNMI, in addition to JTI.
```

Comment:
The OpenConfig integration now supports gRPC, in addition to JTI.

Reply:
Yeah, even JTI is gRPC based so I think original statement seems to be fine.

Reply:
Works for me.